src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,40 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int totalSupply = 0;
+        int totalBuy = 0;
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String value = reader.readLine();
+            while (value != null) {
+                String[] valueParts = value.split(",");
+                if (valueParts[0].trim().equalsIgnoreCase("buy")) {
+                    totalBuy += Integer.parseInt(valueParts[1]);
+                } else if (valueParts[0].trim().equalsIgnoreCase("supply")) {
+                    totalSupply += Integer.parseInt(valueParts[1]);
+                }
```

Comment:
The 'trim()' method is being called repeatedly on the same string. Consider storing the trimmed value in a local variable to avoid redundant operations.